asab/library/providers/zookeeper.py

```diff
@@ -98,6 +104,20 @@ class ZooKeeperLibraryProvider(LibraryProviderABC):
 	If `path` from [zookeeper] section is missing, an application class name will be used
 	Ex. `/BSQueryApp/library`
 
+	Usage of TenantContextVar:
+		The 'TenantContextVar' context variable is used to specify the current tenant context.
+		It can be set at the beginning of an operation (e.g., a web request) to ensure that all
+		subsequent operations within that context are executed under the specified tenant.
+		The context variable is reset to its default value (None) at the end of the operation.
+
+	Example:
+	```
+	tenant_token = TenantContextVar.set('tenant123')
```

Comment:
Use the full ASAB path for this so that it is the most illustrative.

Reply:
@ateska I am little confused.
How do you mean?
**`Use the full ASAB path for this so that it is the most illustrative.`**